task_01.rst

```diff
@@ -0,0 +1,56 @@
+
+Why I Love Python
```

Comment:
It's a good attempt but you're missing all of your headers. The way to create a header is to pick a character and repeat it below and/or above the heading text, eg:

```
not a header
```

```
========
a header
========

lower header
------------
```

It's important that the header character in the over / underline be the exact same number of characters in the heading text.

For each heading type it encounters (Eg, if it encounters over/under `=` first), it will assume it's subordinate to the prior header. So, if no header was specified before it, it assumes it's the highest level or `h1` header. The next type encountered (eg, `-`) would be one level under that aka an `h2`. The next `-` it encounters will also be an `h2` and a new header type, like an `*` would be come the next level down (h3) and so on.